### PR TITLE
Add MostPopularFood component and roundedButton

### PR DESCRIPTION
**Pull request by r-kundan**

### CODE REVIEW

File: src/components/MostPopularFood.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            </svg>
+          </div>
+
+          <div className="flex flex-col justify-center items-center pt-16 gap-5">
+            <h2 className="text-3xl text-wrap">Sweet Corn Soup</h2>
+            <h5 className="text-2xl">Flat Rs.100 off</h5>
+            <h6 className="text-3xl">&#8377;119</h6>
+          </div>
+        </div>
+      </div>
```
Comment (ketanbaitule) [inline src/components/MostPopularFood.jsx:68]: Convert this to different Component.

**Review by ketanbaitule — CHANGES_REQUESTED** [review]

File: src/components/MostPopularFood.jsx (hunk 1 above)
Comment (ketanbaitule) [reply src/components/MostPopularFood.jsx:68]: MostPopularFood.tsx keep this file `app` dir

File: src/components/ProductItem.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                <path d="M316.9 18C311.6 7 300.4 0 288.1 0s-23.4 7-28.8 18L195 150.3 51.4 171.5c-12 1.8-22 10.2-25.7 21.7s-.7 24.2 7.9 32.7L137.8 329 113.2 474.7c-2 12 3 24.2 12.9 31.3s23 8 33.8 2.3l128.3-68.5 128.3 68.5c10.8 5.7 23.9 4.9 33.8-2.3s14.9-19.3 12.9-31.3L438.5 329 542.7 225.9c8.6-8.5 11.7-21.2 7.9-32.7s-13.7-19.9-25.7-21.7L381.2 150.3 316.9 18z" />
+              </svg>
+            </div>
+
+            <div className="flex flex-col justify-center items-center pt-16 gap-5">
+              <h2 className="text-3xl text-wrap">Sweet Corn Soup</h2>
+              <h5 className="text-2xl">Flat Rs.100 off</h5>
+              <h6 className="text-3xl">&#8377;119</h6>
+            </div>
+          </div>
```
Comment (ketanbaitule) [inline src/components/ProductItem.jsx:88]: We only want this much in this file.

Take their data from props

File: src/app/MostPopularFood.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import ProductItem from "@/components/ProductItem"
+
+function MostPopularFood() {
+  return (
+    <>
+       <div className="flex justify-between">
+        <SecondaryHeading>Most Popular Food</SecondaryHeading>
+        <RoundedButton text={"See more"} />
+      </div>
+        <ProductItem/>
```
Comment (ketanbaitule) [inline src/app/MostPopularFood.tsx:13]: Pass data in this element via props

And repeat this

**Review by ketanbaitule — CHANGES_REQUESTED** [review]

File: src/app/MostPopularFood.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+function MostPopularFood() {
+  return (
+    <>
+       <div className="flex justify-between">
+        <SecondaryHeading>Most Popular Food</SecondaryHeading>
+        <RoundedButton text={"See more"} />
+      </div>
+
+      <div className="grid grid-cols-3 gap-4">
```
Comment (ketanbaitule) [inline src/app/MostPopularFood.tsx:14]:
```suggestion
      <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
```

For responsiveness

**Review by ketanbaitule — CHANGES_REQUESTED** [review]

File: src/app/MostPopularFood.tsx (hunk 4 above)
Comment (r-kundan) [reply src/app/MostPopularFood.tsx:14]: ok

**Review by ketanbaitule — APPROVED** [review]

**Pull request merged**